src/redux/configureStore.js

```diff
@@ -0,0 +1,9 @@
+import { combineReducers, createStore, applyMiddleware } from 'redux';
+import logger from 'redux-logger';
+import books from './books/books';
+import categories from './categories/categories';
+
+const rootReducer = combineReducers({
+  books, categories,
+});
+export default createStore(rootReducer, applyMiddleware(logger));
```

Comment:
- [ ] **[OPTIONAL]** Kindly declare a variable `store` and set it to the value of ` createStore(rootReducer, applyMiddleware(logger))` and then export the store as default export. see suggestion below;

```js
const store = createStore(rootReducer, applyMiddleware(logger));

export default store;
```